File: php/Bolo.php
<?php
    class Bolo{
    public $cobertura;
    public $recheio;
    public $tamanho;
        public function listarPartes():void{
            echo "Tamanho do Bolo: $this->tamanho, cobertura: $this->cobertura e recheio: $this->recheio";
        } 
        
    }

File: java/Bolo.java
public class Bolo{
    
    private String cobertura;
    private String recheio;
    private String tamanho;
    
    public void setCobertura(String cobertura) {
        this.cobertura= cobertura;
    }
    
    public void setRecheio(String recheio) {
        this.recheio = recheio;
    }
    
    public void setTamanho(String tamanho) {
        this.tamanho = tamanho;
    }
    
    @Override
    public String toString() {
        return "Tamanho do Bolo: " + tamanho + ", cobertura: " + cobertura + ", recheio: " + recheio;
    }


}